README.md

```diff
@@ -35,6 +35,12 @@ manually in the `psql` flags of the commands that follow. You should also make
 sure to create the `postgres` user with the necessary permissions to create a
 new database.
 
+If you don't have Postgres or PgBench installed, install them:
+
+```shellsession
+sudo yum install -y postgresql15 postgresql15-contrib
```

Comment:
We almost definitely want to give install steps for more than just RHEL-based systems

Reply:
I agree, but I think people should be able to modify for the package manager of their choice.

Reply:
I think we should reword this differently to avoid "defaulting" to `yum` as the package manager, then - maybe something more english-sentence like?

Reply:
think eg something like the list here: https://github.com/mit-pdos/noria#running-noria